feature-libs/checkout/scheduled-replenishment/components/services/checkout-replenishment-form.service.ts

```diff
@@ -64,8 +62,6 @@ export class CheckoutReplenishmentFormService implements OnDestroy {
         this.eventService.get(RestoreSavedCartSuccessEvent),
         this.eventService.get(PaymentDetailsCreatedEvent),
         this.eventService.get(PaymentDetailsSetEvent),
-        this.eventService.get(OrderPlacedEvent),
-        this.eventService.get(ReplenishmentOrderScheduledEvent),
```

Comment:
I guess this was a mistake to include the repl event here, and then do `next` with `ORDER_TYPE.PLACE_ORDER`.
But why are we removing `OrderPlacedEvent`?

Reply:
Nope

as mentioned, OrderPlacedEvent by default is using order_type.place_order, therefore no need to 'reset'